keyboards/kbdfans/kbd6x/keymaps/mekberg/keymap.c

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2018 MechMerlin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// #define MY_LOCK C(A(KC_L))      // Mac: Lock
+#define MY_LOCK C(LCMD(KC_L))      // Mac: Lock (when Alt/Command have been swapped in macOS)
```

Comment:
Just a note, neither of these combinations work for me (macOS 10.15, no Karabiner installed or anything). The default shortcut seems to be Ctrl+Command+Q.

Reply:
Well, they're labeled "MY_" for a reason :)

They trigger a custom key-combo in BTT. It activates the screensaver directly (which in turn requires password immediately). I use this instead of the built-in lock because it disables my external screen, which then makes the Macbook sleep when its closed.

I can expand on the comments to clarify this, but otherwise it works exactly as it should.

Reply:
Ah, no worries. That's why I makred this as a comment rather than a change request, I figured it might be something like that 😄